src/com/yetanalytics/lrs_admin_ui/views/status.cljs

```diff
@@ -0,0 +1,227 @@
+(ns com.yetanalytics.lrs-admin-ui.views.status
+  (:require
+   [reagent.core :as r]
+   [re-frame.core :refer [subscribe dispatch]]
+   [clojure.string :as cs]
+   [com.yetanalytics.lrs-admin-ui.functions :as fns]
+   [com.yetanalytics.lrs-admin-ui.views.status.vis :as vis]
+   [goog.string :refer [format]]
+   [goog.string.format]))
+
+(defn big-number
+  [vis-key label sub-qvec]
+  (let [value @(subscribe sub-qvec)]
+    [:div.big-number
+   [:div.big-number-value
```

Comment:
Huh TIL you can write div classes like this in Hiccup.

Reply:
Yep, a la [HAML](https://haml.info/). You can also use `#id` for ids